cmd/embedded-cluster/restore.go

```diff
@@ -561,12 +605,24 @@ var restoreCommand = &cli.Command{
 		}
 
 		logrus.Debugf("restoring infra from backup %q", backup.Name)
-		if err := restoreFromBackup(c.Context, backup, DisasterRecoveryComponentInfra); err != nil {
+		if err := restoreFromBackup(c.Context, backup, DisasterRecoveryComponentInfra, "", nil); err != nil {
 			return err
 		}
 
+		if c.String("airgap-bundle") != "" {
+			logrus.Debugf("restoring embedded cluster registry from backup %q", backup.Name)
+			err := restoreFromBackup(c.Context, backup, DisasterRecoveryComponentChart, "registry", &metav1.LabelSelector{
```

Comment:
how about just adding a new `DisasterRecoveryComponentChartRegistry` type and keep the logic of determining the label selector in the `restoreFromBackup` function? that way, you also wouldn't have to pass the two extra parameters.